docs/eventing/samples/kafka/resetoffset/README.md

```diff
@@ -1,91 +1,83 @@
----
-title: "Apache Kafka ResetOffset Example"
-linkTitle: "ResetOffset Example"
-weight: 20
-type: "docs"
----
-
-Kafka backed Channels are potentially different from other implementations in
-that they provide temporal persistence of events. This capability allows for
-certain advanced use cases to be supported, including the ability to "replay"
-prior events via repositioning the current location in the event-stream.  This
-capability can be extremely useful when attempting to recover from unexpected
-Subscriber downtime.
-
-The
-[ResetOffset CRD](https://github.com/knative-sandbox/eventing-kafka/tree/main/config/command/resetoffset)
-exposes the ability to manipulate the location of the ConsumerGroup Offsets in
-the event stream of a given Knative Subscription. Without the ResetOffset CRD
-you would be responsible for manually stopping ConsumerGroups and manipulating
-Offsets.
+# ResetOffset Example
 
-!!! note
-    Repositioning the ConsumerGroup Offsets will impact the event ordering and
-    is intended for failure recovery scenarios. This capability needs to be used
-    with caution only after reviewing the CRD documentation linked above.
+Kafka-backed Channels differ from other Channel implementations because they provide temporal persistence of events.
+
+Event persistence enables certain advanced use cases to be supported, such as the ability to replay prior events by repositioning their current location in the event stream. This feature is useful when attempting to recover from unexpected Subscriber downtime.
+
+The [ResetOffset custom resource definition (CRD)](https://github.com/knative-sandbox/eventing-kafka/tree/main/config/command/resetoffset) exposes the ability to manipulate the location of the ConsumerGroup Offsets in the event stream of a given Knative Subscription. Without the ResetOffset CRD, you must manually stop ConsumerGroups and manipulating Offsets.
 
 !!! note
     ResetOffsets are currently only supported by the Distributed KafkaChannel
-    implementation. It is expected that the Consolidated KafkaChannel
-    implementation will support them in the near future.
```

Comment:
@travis-minke-sap we don't really talk about what might happen in the future in docs, just FYI 🙂

Reply:
👍